src/calculateRentalCost.js

```diff
@@ -5,6 +5,16 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
```

Comment:
Prefer 'const' over 'let' where possible to avoid unintentional changes. Since 'totalCost' is not reassigned, use 'const'.